.github/workflows/desmos_bindings.yml

```
@@ -29,25 +43,40 @@ jobs:
     name: Upload coverage
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
+      - name: Checkout 🛎️
         uses: actions/checkout@v2
 
-      - name: Install stable toolchain
+      - name: Verify .rs files 👀
+        uses: technote-space/get-diff-action@v6.1.0
+        id: git_diff
+        with:
+          PATTERNS: |
+            **/**.rs
```

```suggestion
          PATTERNS: |
            **/**.rs
            Cargo.lock
            Cargo.toml
```
It should include Cargo.toml and Cargo.lock.